### ISSUE TITLE
Strange feeling with Gaomon S620

### ISSUE THREAD

**Issue by Nykreuz** [InfinityGhost/TabletDriver#29]
## Hardware/software information

Tablet Driver version: 0.3.0

Tablet model: Gaomon S620

Hi, i got my gaomon working correctly with those drivers but when i play osu! it feels like smoothing is increasing and increasing constantly btw, i dont have any smoothing filters. My head is break trying to play comfortably osu! With this tablet, only problems and problems. Cursor accuracy has problem too...

**Comment by xCuri0** [comment]
did you try using anti smoothing filter ?

**Comment by Nykreuz** [comment]
Yes, i got it enable, and it fix part of problem but still cant do jumps on osu!. Just changed it from wacom ctl-4100 if i can remember correctly the name only for better RPS and LPI. Drawing i have 0 problem and all is fine.

**Comment by Nykreuz** [comment]
sliders feel like they have some glue on it or something and cursor move like 1x and when it ends cursor back to normal sensibility that is 1.35x and then goes to fuck a corner by me for that...

**Comment by InfinityGhost** [comment]
What settings are you running? What input mode, etc?

**Comment by Nykreuz** [comment]
Using pen as Mouse and setting for Game. Only enable anti smoothing filter with 0.500 shape and 5ms compensation. It moves comfortable now but still do that sliders things.

**Comment by InfinityGhost** [comment]
Try the latest release to see if this is still present.
> https://github.com/InfinityGhost/TabletDriver/releases/tag/v0.3.1a

**Comment by Berkkyyyy** [comment]
i tryed the latest versionv0.3.1a) and my cursor didnt move help

**Comment by Nykreuz** [comment]
it feels better on slider but still something strange on how it feels.

**Comment by Berkkyyyy** [comment]
it says Unkown Huion Tablet!Check startuplog.txt

version: v.0.3.0

**Comment by Nykreuz** [comment]
glue on slider still there, making hard jumps after a slider. Long slider literally cant be done, i move out of tablet.

**Comment by Berkkyyyy** [comment]
ok i got 3.1 but my cursor doesnt move although i installed the vmulti driver

**Comment by xCuri0** [comment]
@Nykreuz does changing your osu keybinds (zx by default) to something else do anything ?

**Comment by Nykreuz** [comment]
@xCuri0 i did it and dont change nothing (first on we, and now on gh)

**Comment by InfinityGhost** [comment]
In any case it just stops not allowing you to move your cursor, reinstall both the Gaomon driver, and the vmulti driver.

**Comment by Nykreuz** [comment]
Using anti-smoother filter and shape of 1.000 it feels better, but again, more time playing = more glue on slider (i dont know how to name that problem with slider hahaha)

**Comment by cillynder** [comment]
This sounds like an issue of the driver not being able to keep up. Is your CPU old by any chance? Is anything else on your computer hogging resources?

**Comment by Nykreuz** [comment]
@MoeLava i have a i7 8700k , 16gb ram and a gtx 1080ti, literally OSU works at 0,29ms and 144hz.

**Comment by Berkkyyyy** [comment]
when i download the tabletdriver it always gets detected as a virus.help

**Comment by Nykreuz** [comment]
@Berkkyyyy just pass it through windows defender

**Comment by Nykreuz** [comment]
@MoeLava and @InfinityGhost i tried to use a Xp Pen g640 that i had without open. Nothing strange on this tablet with same settings as Gaomon S620

**Comment by InfinityGhost** [comment]
> when i download the tabletdriver it always gets detected as a virus.help

#16, its safe.

**Comment by Berkkyyyy** [comment]
> > when i download the tabletdriver it always gets detected as a virus.help
> 
> #16, its safe.

i know but it doesnt even let me download

**Comment by Berkkyyyy** [comment]
ah now it does download xd thx

**Comment by Nykreuz** [comment]
guys, literally i have shape on 4.500 and it feel much better xD (" value around 0.5 recommended") ilogic.

**Comment by Berkkyyyy** [comment]
@Nykreuz what do you have at velocity and compensation?

**Comment by Nykreuz** [comment]
0 on both

**Comment by Berkkyyyy** [comment]
And what preset do you have?

**Comment by Nykreuz** [comment]
no preset

**Comment by UntitledShoe** [comment]
I have a gaomon s620 and I got the drivers to download and it is finding the tablet but the cursor won't move. Any ideas? (it also might just be my settings)
